packages/react-scripts/template/src/utils/Request.js

```diff
@@ -0,0 +1,66 @@
+import URI from 'urijs';
```

Comment:
we should move this `elmer`, even though its not a good place for this

Reply:
Request can be custom. For example the nimble one has some stuff that isn't needed by vmware. It's basically just a pattern that makes things easy.